tests/utils/constants.py

```diff
@@ -74,4 +74,23 @@
         ),
         "chainId": CHAIN_ID,
     },
+    {
+        "type": 2,
+        "gas": 100_000,
+        "maxFeePerGas": 2_000_000_000,
+        "maxPriorityFeePerGas": 2_000_000_000,
+        "data": "0x616263646566",
+        "nonce": 34,
+        "to": "",
```

Comment:
is `"to": ""` the same as `"to": ADDRESS_ZERO`?

Reply:
I remember @Eikix saying during the hackathon that `""` was what they would be sending via the RPC, which is why I used it. But yes, `""` and `ZERO_ADDRESS` both result in `to` being **0** inside the contract.

Reply:
very cool !